packages/libllvm/build.sh

```diff
@@ -114,6 +131,11 @@ termux_step_post_make_install() {
 	for tool in clang clang++ cc c++ cpp gcc g++ ${TERMUX_HOST_PLATFORM}-{clang,clang++,gcc,g++,cpp}; do
 		ln -f -s clang-${TERMUX_PKG_VERSION:0:2} $tool
 	done
+
+	cd $TERMUX_PREFIX/lib/clang/$TERMUX_PKG_VERSION/lib/
+	# Trying to build a program with -fsanitizer=address on device leads to an error
+	# due to clang looking for the libs in linux/ and not android/
+	mv android/ linux/
```

Comment:
Hmm, this is because of a bug in clang, which you can patch instead of using this hack. From my local patches that I never cleaned up and submitted:
```
diff --git a/clang/lib/Driver/ToolChain.cpp b/clang/lib/Driver/ToolChain.cpp
index 469ef62f1a9d..218f5af748f4 100644
--- a/clang/lib/Driver/ToolChain.cpp
+++ b/clang/lib/Driver/ToolChain.cpp
@@ -374,7 +374,7 @@ StringRef ToolChain::getOSLibName() const {
   case llvm::Triple::Solaris:
     return "sunos";
   default:
-    return getOS();
+    return Triple.isAndroid() ? "android":getOS();
   }
 }
 
```

Reply:
Thanks, I'll add it to this PR!